test/sagemaker_tests/pytorch/training/resources/mnist/mnist.py

```diff
@@ -17,6 +17,7 @@
 opener = urllib.request.build_opener()
 opener.addheaders = [('User-agent', 'Mozilla/5.0')]
 urllib.request.install_opener(opener)
+from packaging import version
```

Comment:
nit: Please use `from packaging.version import Version`, as the `Version` class is more intuitive in comparing version strings than `version.parse`.